docs/samples.md

```diff
@@ -83,3 +84,5 @@ a.volume -= 10.0
 It would be difficult to know what version to apply to each:
 
 ![Version inconsistency](./img/samples-04.png)
+
+Therefore we have decided that any time a sample is split a new sample entity must be created for the new aliqute. While, when a property is simply changed the sample identity remains the same.
```

Comment:
Cool, I'll remove the "we have decided" into a more "this is how it is" statement.

Reply:
:+1: